tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Proxy/AwsApiGatewaySpanFactory.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright file="AwsApiGatewaySpanFactory.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Proxy;
+
+/// <summary>
+/// Creates spans representing requests handled by AWS API Gateway.
+/// </summary>
+internal class AwsApiGatewaySpanFactory : IInferredSpanFactory
+{
+    private const string OperationName = "aws.api-gateway";
```

Comment:
This is incorrect see: https://github.com/DataDog/dd-trace-dotnet/pull/6624#discussion_r1961952382
Unsure how I got that `-` in there 😅 
```suggestion
    private const string OperationName = "aws.apigateway";
```